tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+SUPPORTED_HYPERVISORS="qemu-sev qemu-snp"
+
+setup_unencrypted_confidential_pod() {
+    local pod_name="unencrypted-confidential"
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
+    export SSH_KEY_FILE="${TEST_DIR}/container-ssh-key"
+
+    if [[ "$SUPPORTED_HYPERVISORS" == *"${KATA_HYPERVISOR}"* ]]; then
```

Comment:
You can drop the if condition here, as we know this will only be called for hypervisors that are part of the supported list, and we ensure that in the bats file.